docker/images/kumascript/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:10.14.2@sha256:09dedfecd09e55b5b39ca9f04c40b1b70889a0b65d2de4d9d510d39b1a213c5f
+FROM node:10.15.3@sha256:c5e919a89352d3ce6a883dde54a5d51dde12229c2d11088593cd1f3efefcc16e
```

Comment:
kumascript/package.json includes an "engines" property that also specifies the Node version, so presumably this change would need to be synchronized with a KumaScript change that updates the engines property.

I don't know what the engines property is good for, however, so maybe it would be better to first do a PR to kumascript to just remove the engines property so that we can do upgrades like this without that kind of coordination.